x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/ScoreFunctionIT.java

```diff
@@ -0,0 +1,413 @@
+/*
```

Comment:
Should we add a test to check if the capability is enable to run these tests only when it is the case?